src/discord-cluster-manager/run_eval.py

```diff
@@ -267,9 +267,6 @@ def run_single_evaluation(
     mode: str,
     tests: Optional[str] = None,
     benchmarks: Optional[str] = None,
-    test_timeout: int = 30,
```

Comment:
These need to stay, as this is where we can override default timeouts once we get larger tasks.